float64[6] motor
